### PR TITLE
Fixed link to main content

### PR DESCRIPTION
**Pull request by ch-rfishwick**
__Short description outlining key changes/additions__
Fixed tab link to main page content

__JIRA Ticket Number__
FA-276

### Type of change

*  Bug fix
* [ ] New feature
* [ ] Breaking change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

File: src/main/resources/templates/layouts/baseLayout.html [hunk 1]
```diff
@@ -11,7 +11,9 @@
       rel="stylesheet"
       th:href="@{{cdnUrl}/stylesheets/chs-styles.css(cdnUrl=${@environment.getProperty('cdn.url')})}"
       type="text/css"/>
-
+  <link
+      href="http://chs-dev.internal:8080/stylesheets/assets-digital-cabinet-office-gov-uk-static/govuk-template.css"
```
Comment (CH-JonathanBlake) [inline src/main/resources/templates/layouts/baseLayout.html]: This doesn't seem right at all.
We can't hard code this to the development server.

File: src/main/resources/templates/fragments/header.html [hunk 2]
```diff
@@ -5,7 +5,6 @@
 </head>
 <body class="govuk-template__body">
 <th:block th:fragment="header">
-  <a class="govuk-skip-link" href="#main-content">Skip to main content</a>
```
Comment (CH-JonathanBlake) [inline src/main/resources/templates/fragments/header.html]: https://www.w3.org/TR/WCAG20-TECHS/G1.html
This is reccommended by W3C, if we're removing it then we ought to replace it with something

File: src/main/resources/templates/layouts/baseLayout.html (hunk 1 above)
Comment (Ngriffiths1) [reply src/main/resources/templates/layouts/baseLayout.html]: same

File: src/main/resources/templates/fragments/header.html (hunk 2 above)
Comment (Ngriffiths1) [reply src/main/resources/templates/fragments/header.html]: same

Comment (ch-rfishwick) [reply src/main/resources/templates/fragments/header.html]: I did, it's in the baseLayout

Comment (CH-JonathanBlake) [reply src/main/resources/templates/fragments/header.html]: Do you mean that hard coded thing that is pointing at a .css?

Comment (ch-rfishwick) [reply src/main/resources/templates/fragments/header.html]: No, the other change to the base layout

Comment (CH-JonathanBlake) [reply src/main/resources/templates/fragments/header.html]: Sorry, I forgot that it wasn't the only change. Deleted the other comment

File: src/main/resources/templates/layouts/baseLayout.html [hunk 3]
```diff
@@ -29,6 +31,11 @@
 <script
     th:src="@{{cdnUrl}/javascripts/vendor/jquery-3.3.1.min.js(cdnUrl=${@environment.getProperty('cdn.url')})}"></script>
 <div class="entire-wrapper">
+  <div id="skiplink-container">
+    <div>
```
Comment (ch-dclarke) [inline src/main/resources/templates/layouts/baseLayout.html]: Why is there a <div> inside an <div> here?

Comment (ch-rfishwick) [reply src/main/resources/templates/layouts/baseLayout.html]: Because that is how it is written on the current site. But it can be removed

Comment (ch-dclarke) [reply src/main/resources/templates/layouts/baseLayout.html]: Please remove it if it serves no purpose.

**Pull request merged**